ci/debian/centreon-gorgone.postinst

```diff
@@ -32,12 +32,29 @@ if [ "$1" = "configure" ] ; then
     /var/lib/centreon-gorgone \
     /var/log/centreon-gorgone
 
+  chown root:root \
+    /usr/local/bin/gorgone_config_init.pl \
+    /usr/local/bin/gorgone_audit.pl \
+    /usr/local/bin/gorgone_install_plugins.pl
+
+  chmod 0755 \
```

Comment:
why 755 and not 750 ?

Reply:
> why 755 and not 750 ?

Because I used the same arguments as from the rpm template:
https://github.com/centreon/centreon-gorgone/blob/develop/packaging/centreon-gorgone.spectemplate#L102